src/main/java/gregtech/api/pipenet/block/BlockPipe.java

```diff
@@ -325,23 +326,28 @@ public boolean onPipeActivated(World world, IBlockState state, BlockPos pos, Ent
             }
         }
 
-        EnumFacing coverSide = ICoverable.traceCoverSide(hit);
-        if (coverSide == null) {
+        EnumFacing coverSide = ICoverable.determineGridSideHit(hit);
+        CoverBehavior coverBehavior = pipeTile.getCoverableImplementation().getCoverAtSide(coverSide);
+        
+        if (coverBehavior == null) {
```

Comment:
This change should not be made, as it was what broke pipe and cable connections recently. See the fix in #1464

Reply:
should be okay now